src/utilities/credentials.jl

```diff
@@ -185,13 +195,32 @@ end
 Retrieve sso-specific details for the given `profile`.
 """
 function _aws_get_sso_credential_details(profile::AbstractString, ini::Inifile)
-    sso_start_url = _get_ini_value(ini, profile, "sso_start_url")
-    sso_account_id = _get_ini_value(ini, profile, "sso_account_id")
-    sso_role_name = _get_ini_value(ini, profile, "sso_role_name"; default_value="default")
-    sso_region = _get_ini_value(ini, profile, "sso_region"; default_value=DEFAULT_REGION)
-
-    # sso cache access token
-    access_token = @mock _sso_cache_access_token(sso_start_url)
+    sso_session = _get_profile_value(ini, profile, "sso_session")
+    sso_account_id = _get_profile_value(ini, profile, "sso_account_id") # optional but we require it
+    sso_role_name = _get_profile_value(ini, profile, "sso_role_name"; default_value="default") # optional but we require it
```

Comment:
Definitely seems to be optional as we have a default:
```suggestion
    sso_role_name = _get_profile_value(ini, profile, "sso_role_name"; default_value="default") # Optional but AWS.jl requires it
```